### PR TITLE
full stops added to 6 lectures

### PR DESCRIPTION
**Pull request by AnjuJoon**

### CODE REVIEW

File: source/rst/dynamic_programming/wald_friedman.rst [hunk 1]
```diff
@@ -80,26 +80,26 @@ officer like Schyler were on the premises, he would see after the first
 few thousand or even few hundred [rounds] that the experiment need not
 be completed either because the new method is obviously inferior or
 because it is obviously superior beyond what was hoped for
-:math:`\ldots` ''
+:math:`\ldots` ''.
```
[the remark this answers is not in the document]
Comment (AnjuJoon) [reply source/rst/dynamic_programming/wald_friedman.rst]: yes you are right! Thanks for the review @arnavs I will make the changes

File: source/rst/dynamic_programming/smoothing.rst [hunk 2]
```diff
@@ -18,42 +18,42 @@ Overview
...
 This lecture describes two types of consumption-smoothing  and tax-smoothing models
 
-* one is in the **complete markets** tradition of Lucas and Stokey :cite:`LucasStokey1983`
+* One is in the **complete markets** tradition of Lucas and Stokey :cite:`LucasStokey1983`.
 
-* the other is in the **incomplete markets** tradition  of Hall :cite:`Hall1978` and Barro :cite:`Barro1979`
+* The other is in the **incomplete markets** tradition  of Hall :cite:`Hall1978` and Barro :cite:`Barro1979`.
 
-*Complete markets* allow a consumer or government to buy or sell claims contingent on all possible states of the world
+* Complete markets* allow a consumer or government to buy or sell claims contingent on all possible states of the world.
```
[the remark this answers is not in the document]
Comment (AnjuJoon) [reply source/rst/dynamic_programming/smoothing.rst]: Actually, there was a space in all the other points above and below it, so I just added a space in this one to make it even! If you want I can revert this change. Thanks!

[the remark this answers is not in the document]
Comment (arnavs) [reply source/rst/dynamic_programming/smoothing.rst]: Gotcha, all good then.

**Comment by arnavs** [comment]
Thanks @AnjuJoon for the change. I think the `\ldots` thing can be deleted (we don't use it on the HTML version anyway, and it doesn't seem to show up?) But this is not a regression, so all good.

**Pull request merged**